### PR TITLE
update spec for the 0.5.0 release

### PR DESCRIPTION
**Pull request by surajssd**
update spec to new release
change the release tag to not include the git commit
add support for bash auto completion

### CODE REVIEW

File: kompose.spec [hunk 1]
```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
+then
+  %{_bindir}/kompose completion zsh > %{_datadir}/zsh/site-functions/kompose;
+else
+  %{_bindir}/kompose completion bash > %{_datadir}/bash-completion/completions/kompose;
+fi
```
Comment (surajssd) [inline kompose.spec]: @dustymabe @cdrage would like your input on this section!

File: kompose.spec [hunk 2]
```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
```
Comment (dustymabe) [inline kompose.spec]: hmm. what does `${BASH+x}` give you? looks like `x` if $BASH is defined and `` if not. What is the benefit of doing this and not `-z "$BASH"` ?

File: kompose.spec (hunk 1 above)
Comment (dustymabe) [reply kompose.spec]: so my thoughts are we should create these files earlier in the process and include them in the rpm. so basically we compile kompose and the use the binary to create these files. then include them in %files. Also, lets create both zsh and bash completion files to be delivered by the rpm.

File: kompose.spec [hunk 3]
```diff
@@ -52,8 +52,8 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        0.4.0
-Release:        0.1.git%{shortcommit}%{?dist}
```
Comment (dustymabe) [inline kompose.spec:56]: can you remove the shortcommit in a 2nd commit? I want this to be a clear change we decided to do and not something that could be confused as a mistake.

File: kompose.spec (hunk 1 above)
Comment (dustymabe) [reply kompose.spec]: so in `%build`, right after `%gobuild` then run `bin/kompose completion bash > file` and then in `%install` do something like `install` file into appropriate location. Under `%files` add them just like we do `%{_bindir}/kompose`

Comment (dustymabe) [reply kompose.spec]: also, add the bash completion in a 2nd commit please.

**Review by dustymabe — CHANGES_REQUESTED** [review]

File: kompose.spec (hunk 3 above)
Comment (surajssd) [reply kompose.spec:56]: done! working on the auto complete part!

File: kompose.spec (hunk 2 above)
Comment (surajssd) [reply kompose.spec]: read about it here http://stackoverflow.com/a/13864829/3848679

Comment (dustymabe) [reply kompose.spec]: yeah but that is an answer to the question `How do I know if a variable is set in Bash?`. We don't care if the variable is set or not, just that it is not empty :). Good to know how to do that, though.

**Comment by surajssd** [comment]
@dustymabe done

**Comment by surajssd** [comment]
I tested it I have to start bash anew for auto complete to work! Needs to solve that.

**Pull request merged**